systemtest/src/test/java/io/strimzi/systemtest/kafka/KafkaST.java

```diff
@@ -394,28 +393,20 @@ void testRemoveComponentsFromEntityOperator() {
         PodUtils.waitUntilPodStabilityReplicasCount(testStorage.getNamespaceName(), KafkaResources.entityOperatorDeploymentName(testStorage.getClusterName()), 0);
     }
 
-    /**
-     * @description This test case verifies that Kafka with persistent storage, and JBOD storage, property 'delete claim' of JBOD storage.
-     *
-     * @steps
-     *  1. - Deploy Kafka with persistent storage and JBOD storage with 2 volumes, both of these are configured to delete their Persistent Volume Claims on Kafka cluster un-provision.
-     *     - Kafka is deployed, volumes are labeled and linked to Pods correctly.
-     *  2. - Verify that labels in Persistent Volume Claims are set correctly.
-     *     - Persistent Volume Claims do contain expected labels and values.
-     *  2. - Modify Kafka Custom Resource, specifically 'delete claim' property of its first Kafka Volume.
-     *     - Kafka CR is successfully modified, annotation of according Persistent Volume Claim is changed afterwards by Cluster Operator.
-     *  3. - Delete Kafka cluster.
-     *     - Kafka cluster and its components are deleted, including Persistent Volume Claim of Volume with 'delete claim' property set to true.
-     *  4. - Verify remaining Persistent Volume Claims.
-     *     - Persistent Volume Claim referenced by volume of formerly deleted Kafka Custom Resource with property 'delete claim' set to true is still present.
-     *
-     * @usecase
-     *  - JBOD
-     *  - PVC
-     *  - volume
-     *  - annotations
-     */
     @ParallelNamespaceTest
+    @TestDoc(
+        description = @Desc("This test case verifies Kafka running with persistent JBOD storage, and configured with the `deleteClaim`  storage property."),
+        steps = {
+            @Step(value = "Deploy Kafka with persistent storage and JBOD storage with 2 volumes, both of which are configured to delete their Persistent Volume Claims on Kafka cluster un-provision.", expected = "Kafka is deployed, volumes are labeled and linked to Pods correctly."),
+            @Step(value = "Verify that labels in Persistent Volume Claims are set correctly.", expected = "Persistent Volume Claims contains expected labels and values."),
+            @Step(value = "Modify Kafka Custom Resource, specifically 'deleteClaim' property of its first Kafka Volume.", expected = "Kafka CR is successfully modified, annotation of according Persistent Volume Claim is changed afterwards by Cluster Operator."),
```

Comment:
would be good to use the same words - CR / Custom Resource at least in one step.